blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientConfig.scala

```diff
@@ -47,7 +47,7 @@ final case class BlazeClientConfig(// HTTP properties
 
                                    // pipeline management
                                    bufferSize: Int,
-                                   customExecutor: Option[ExecutorService],
+                                   customExecutionContext: Option[ExecutionContext],
```

Comment:
`customExecutionContext` was so we knew if it was a default, and that we have to shut it down ourselves.  What if we eliminate the option and just use `ExecutionContext.global` as the default?